samples/maps/series-tiledwebmap/europe-timezones-twm/demo.js

```diff
@@ -0,0 +1,106 @@
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/europe.topo.json'
+    ).then(response => response.json());
+
+    const { colors } = Highcharts.getOptions();
+
+    // Instantiate the map
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology,
+            spacingBottom: 20
+        },
+
+        title: {
+            text: 'Europe time zones on TiledWebMap'
+        },
+
+        accessibility: {
+            series: {
+                descriptionFormat: 'Timezone {series.name} with {series.points.length} countries.'
+            },
+            point: {
+                valueDescriptionFormat: '{point.name}.'
+            }
+        },
+
+        legend: {
+            enabled: false
+        },
```

Comment:
Yes, it is disabled, because it would need extra logic to create dots in the legend items due to `color: transparent` and set only `borderColor` for the series. Do you think legend is necessary?

Reply:
I think the appropriate way to do this would be to use a `mapline` series with a `color` setting instead of a `map` series with a `borderColor` and transparent `color`.